shell/src/main/java/alluxio/cli/fs/command/AbstractFileSystemCommand.java

```diff
@@ -51,6 +51,17 @@ protected void runPlainPath(AlluxioURI plainPath, CommandLine cl)
       throws AlluxioException, IOException {
   }
 
+  /**
+   * Process the header of the command. Our input path may contain wildcard
+   * but we only want to print the header for once.
+   *
+   * @param cl object containing the original commandLine
+   * @throws AlluxioException
+   * @throws IOException
```

Comment:
avoid empty javadoc `@throws`

Reply:
Thanks! Removed all the empty javadoc `@throws`